pkg/endpoint/policy.go

```diff
@@ -919,6 +917,7 @@ func (e *Endpoint) startRegenerationFailureHandler() {
 			}
 			return fmt.Errorf("regeneration recovery failed")
 		},
+		RunInterval:            1 * time.Second,
```

Comment:
Does this mean that under good conditions, the controller will constantly run just to check the channel and return, ie noop?